### ISSUE TITLE
Project Review

### ISSUE THREAD

**Issue by SidTheEngineer** [natalyakuznet/prj-rev-bwfs-dasmoto#1]
### Criteria 1: HTML and CSS Linking  
* _Score Level:_ 4/4
* _Comment(s):_ CSS file is appropriately linked to the HTML using a relative path
 
### Criteria 2: Implementation of Design Specification and Content
* _Score Level:_ 4/4
* _Comment(s):_ Although the styling of the site doesn't match the design specification, I think it's set up very well and displays the appropriate skillset to really nail the project. Nice work here! The only recommendation I would have for improvement with respect to design would be making sure the links in the header contrast a bit more with the background so that the text is a bit easier to see.
 
### Criteria 3: HTML Elements and Content
* _Score Level:_ 4/4
* _Comment(s):_ Your HTML looks great! I didn't see any issues with syntax/formatting and the file is clean, organized, and concise.
 
### Criteria 4: CSS Selectors and Syntax
* _Score Level:_ 4/4
* _Comment(s):_ Your CSS selectors and syntax looked solid as well, I didn't see any glaring issues here. One thing to be weary of is leaving commented out code throughout a file, as it can eventually clutter the file and make it a bit harder to work through.
 
### Overall Score: 16/16 🔥 
Awesome work! You nailed this project.

**Comment by natalyakuznet** [comment]
I agrree on your viewing of the links in the header, refering  to Creteria Two. When I'd viewed it before the submition, it was as you could see in Safari now, the links actually has to be under the cover.  Google Chrome windows can be different in size, I dont remember its exact size when the links were placed under the cover. But you can manipulate it to see where they were meant to be. Thank you for your evaluation and tips!

**Comment by natalyakuznet** [comment]
I agree on your viewing of the links in the header, referring to Criteria Two. When I'd viewed it before the review, it was as you could see it in Safari now, the links actually have to be under the cover. Google Chrome windows can be different in size, I don't remember its exact size when the links were placed under the cover. But you can manipulate it to see where they were meant to be. Thank you for your evaluation and tips!

**Comment by natalyakuznet** [comment]
I agree on your viewing of the links in the header, referring to Criteria Two. Before the review, I'd viewed it as you could see it in Safari (default), the links actually have to be under the cover. Google Chrome windows can be different in size, I don't remember its exact size (but I believe it should have been its default) when the links were placed under the cover. You can manipulate it to see where they were meant to be. Thank you for your evaluation and tips!
